### ISSUE TITLE
Decoupling not working.

### ISSUE THREAD

**Issue by nextjr** [linuxgurugamer/REKT#8]
The decouplers on the type-w364 inline parts are not working properly, as well as the shroud not disappearing when the oatv decouples from the type-w364 docking ring, love this mod always one of my favorites!

**Comment by linuxgurugamer** [comment]
Umm, first, please post things this on the forum in the thread.  Second, you are saying it’s broken without providing any information.  At the least, the Player.log.
Be specific as to what is happening.  A craft file would also be useful

**Comment by nextjr** [comment]
oh my goodness sorry i didnt know i usually post bugs on github.

**Comment by linuxgurugamer** [comment]
Not a problem.  The reason i ask that you try the forums first is that quite often its not a bug, and others can help on the forums.

**Comment by linuxgurugamer** [comment]
Go ahead and post the info i asked for, ill look at it in the next day or so

**Comment by nextjr** [comment]
I’m not sure how to share the log as it is too big, as well as what kind of craft file would you want just a craft with the problem I assume?

**Comment by linuxgurugamer** [comment]
If you can't paste it here, try the following:

1.  Zip it up and paste the zip
2. Put it on a file sharing site such as dropbox, GoogleDrive or any other.  googledrive is sometimes a problem regarding permissions, though.
3. The craft that you are having problems with.  The log file will show me the mods needed.

**Comment by nextjr** [comment]
[Player.zip](https://github.com/linuxgurugamer/REKT/files/5745061/Player.zip)

[smthn.zip](https://github.com/linuxgurugamer/REKT/files/5745062/smthn.zip)

**Comment by linuxgurugamer** [comment]
And can you please describe in detail the symptoms?  A video would also be helpful, if possible.

**Comment by nextjr** [comment]
https://drive.google.com/file/d/1ZzpzAAcabmvQqCTYv77qeaiM2bM2z_gM/view?usp=sharing

**Comment by linuxgurugamer** [comment]
I'm looking into this now.  Do you think you could make a simpler vessel to show the problem?

**Comment by linuxgurugamer** [comment]
While a simple vessel would be nice, I've been able to replicate it, and have found a number of problems with various nodes.  I'm working on it, not sure how long this will take

**Comment by nextjr** [comment]
Take your time! I just love this mod and want to see it improve!

~Next

**Comment by linuxgurugamer** [comment]
Ok, I think the main problem is that KSP can't have two docking ports on a single part, which is what the OTAV Double is trying to do.  It will work with the (fixed) decoupler, but not as a docking port.
Still working on it.  
Edit: I may end up removing that part

**Comment by nextjr** [comment]
﻿
﻿Which one will be the docking port then? Because personally I’d rather have the docking pads connect via docking port and have the canopy just be the decoupler.

Also could the docking pads themselves be a separate part that acts as a decoupler as well? You’d have to assemble it before using it as a sub assembly but it would work right?

~Next

**Comment by linuxgurugamer** [comment]
I'm having problems working with these models, so not sure of the final solution

**Comment by nextjr** [comment]
Alright just let me know when/if you get it fixed!

~Next

**Comment by nextjr** [comment]
Just wondering if you've come any closer to the solution. Again I'm very glad you're even taking the time to look at this problem for me.

**Comment by linuxgurugamer** [comment]
No, and sorry to say, for now, they are on the back burner.  Not sure if/when it can be fixed, I may try to find someone to take a look at the models

**Comment by nextjr** [comment]
Alright thank you I hope it gets fixed eventually even if i have to wait a few years! :D
